packages/react-reconciler/src/ReactFiberCommitWork.js

```diff
@@ -881,7 +881,6 @@ function unmountHostComponents(current): void {
         }
         parent = parent.return;
       }
-      currentParentIsValid = true;
```

Comment:
Every switch case we do `break`, so this line is unreachable

Reply:
This looks reachable to me. The `break` takes us to exactly this line.